File: Arrays/Easy/largestElement/largestElement.py
# brute force 

# def largest(arr):
#     return max(arr)

# def largest(arr):
#     arr.sort()
#     return arr[len(arr)-1]

# def largest(arr):
#     arr.sort()
#     return arr[-1]

#optimal approach
def largest(arr):
    maxi=arr[0]
    for i in range(1,len(arr)):
        if arr[i]>maxi:
            maxi=arr[i]
        
    return maxi

a=[1,54,76,87,99,2]
print(largest(a))

File: Arrays/Easy/largestElement/largestElement.js
//brute force
// const largest=(arr)=>{
//     arr.sort((a,b)=>a-b);
//     return arr[arr.length-1]
// }

//optimal approach:-
const largest=(arr)=>{
    let maxi=arr[0];
    let n=arr.length;
    for(let i=0;i<n;i++){
        if(arr[i]>maxi){
            maxi=arr[i];
        }
    }
    return maxi;
}

a=[2,5,7,9,8,75];

console.log(largest(a));